### PR TITLE
Adds ar, cs, ko, and viwiki.  Minor fixes to enwiki models. 

### PR DESCRIPTION
**Pull request by halfak**
We got a major improvement here by converting all of the word to lower case before looking up the vectors.

### CODE REVIEW

**Comment by halfak** [comment]
Replaces https://github.com/wikimedia/drafttopic/pull/45

File: drafttopic/utilities/add_central_africa.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+"""
+Adds "Geography.Regions.Africa.Central Africa" to the labels manually.
+This is necessary to deal with a gap in the WikiProject page matching
+logic.   For various reasons, very few pages get tagged as being
+topical to Central Africa.  This script ensures that at least *some*
+articles get tagged as topically relevant.
+
```
Comment (accraze) [inline drafttopic/utilities/add_central_africa.py]: helpful docs :+1:

File: model_info/enwiki.drafttopic.md [hunk 2]
```diff
@@ -18,944 +18,944 @@ Model Information:
...
-		Culture.Visual arts.Visual arts*             0.991
-		Geography.Geographical                       0.992
-		Geography.Regions.Africa.Africa*             0.996
-		Geography.Regions.Africa.Central Africa      1
+		Culture.Visual arts.Visual arts*             0.992
+		Geography.Geographical                       0.994
+		Geography.Regions.Africa.Africa*             0.997
+		Geography.Regions.Africa.Central Africa      0.893
 		Geography.Regions.Africa.Eastern Africa      1
 		Geography.Regions.Afr
```
Comment (accraze) [inline model_info/enwiki.drafttopic.md]: nice improvement for `pr_auc`

File: model_info/kowiki.articletopic.md [hunk 3]
```diff
@@ -0,0 +1,961 @@
...
+		STEM.Engineering                             0.984
+		STEM.Libraries & Information                 0.987
+		STEM.Mathematics                             0.988
+		STEM.Medicine & Health                       0.987
+		STEM.Physics                                 0.989
+		STEM.STEM*                                   0.981
+		STEM.Space                                   0.993
+		STEM.Technology                              0.981
+		-------------------------------------------  -----
+	pr_auc (micro=0.836, macro=0.725):
```
Comment (accraze) [inline model_info/kowiki.articletopic.md]: solid `pr_auc`

File: model_info/arwiki.articletopic.md [hunk 4]
```diff
@@ -0,0 +1,961 @@
...
+		STEM.Engineering                             0.981
+		STEM.Libraries & Information                 0.978
+		STEM.Mathematics                             0.985
+		STEM.Medicine & Health                       0.979
+		STEM.Physics                                 0.987
+		STEM.STEM*                                   0.977
+		STEM.Space                                   0.993
+		STEM.Technology                              0.976
+		-------------------------------------------  -----
+	pr_auc (micro=0.784, macro=0.641):
```
Comment (accraze) [inline model_info/arwiki.articletopic.md]: pretty good

File: model_info/cswiki.articletopic.md [hunk 5]
```diff
@@ -0,0 +1,961 @@
...
+		STEM.Engineering                             0.985
+		STEM.Libraries & Information                 0.977
+		STEM.Mathematics                             0.992
+		STEM.Medicine & Health                       0.983
+		STEM.Physics                                 0.988
+		STEM.STEM*                                   0.981
+		STEM.Space                                   0.994
+		STEM.Technology                              0.981
+		-------------------------------------------  -----
+	pr_auc (micro=0.83, macro=0.721):
```
Comment (accraze) [inline model_info/cswiki.articletopic.md]: :+1:

File: model_info/enwiki.articletopic.md [hunk 6]
```diff
@@ -18,944 +18,944 @@ Model Information:
...
 		Geography.Regions.Africa.Western Africa      0.999
-		Geography.Regions.Americas.Central America   0.998
-		Geography.Regions.Americas.North America     0.977
-		Geography.Regions.Americas.South America     0.997
-		Geography.Regions.Asia.Asia*                 0.985
+		Geography.Regions.Americas.Central America   0.999
+		Geography.Regions.Americas.North America     0.979
+		Geography.Regions.Americas.South America     0.998
+		Geography.Regions.Asia.Asia*                 0.989
 		Geogr
```
Comment (accraze) [inline model_info/enwiki.articletopic.md]: awesome

File: model_info/viwiki.articletopic.md [hunk 7]
```diff
@@ -0,0 +1,961 @@
...
+		STEM.Engineering                             0.987
+		STEM.Libraries & Information                 0.967
+		STEM.Mathematics                             0.988
+		STEM.Medicine & Health                       0.983
+		STEM.Physics                                 0.988
+		STEM.STEM*                                   0.983
+		STEM.Space                                   0.995
+		STEM.Technology                              0.982
+		-------------------------------------------  -----
+	pr_auc (micro=0.841, macro=0.721):
```
Comment (accraze) [inline model_info/viwiki.articletopic.md]: nice :+1:

**Pull request merged**